frontends/web/package.json

```diff
@@ -18,18 +18,18 @@
     "build/*"
   ],
   "devDependencies": {
-    "@types/jest": "23.3.2",
+    "@types/jest": "^23.3.10",
```

Comment:
Why do we introduce now a `^`?

Reply:
good catch, because I didn't check.

`yarn upgrade @types/jest ts-jest --latest`